tests/session/session-getTransactionOptions-001.phpt

```diff
@@ -0,0 +1,60 @@
+--TEST--
+MongoDB\Driver\Session::getTransactionOptions()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto(); ?>
+<?php skip_if_no_transactions(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+
+// Pass in options that will be overwritten by the transaction-specific options later on
+$session = $manager->startSession(['defaultTransactionOptions' => [
+    'writeConcern' => new \MongoDB\Driver\WriteConcern('majority'),
+    'readConcern' => new \MongoDB\Driver\ReadConcern('local'),
+    'readPreference' => new \MongoDB\Driver\ReadPreference('primaryPreferred'),
+    'maxCommitTimeMS' => 2,
+]]);
+
+var_dump($session->getTransactionOptions());
+
+$session->startTransaction([
```

Comment:
Should we have one test with no transactions options being used, to assert that `getTransactionOptions()` does not create keys for unset options?